sql_metadb/derived_tables/po_notes

```diff
@@ -0,0 +1,31 @@
+--MetaDB: Table po_notes
```

Comment:
See the directive syntax at: https://metadb.dev/doc/#_metadbtable

Also note that the name is "Metadb" not "MetaDB".